app.js

```diff
@@ -4,7 +4,8 @@ const cors = require('cors')
 
 const contactsRouter = require('./routes/api/contacts')
 
-const app = express()
+const app = express() // веб-сервер
```

Comment:
```suggestion
const app = express();
``` коменти без зайвої необхідності краще не писати

Reply:
перевірте чи у вас підключений prettier та чи він відпрацьовує